<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissionCompilerMainWindow</class>
 <widget class="QMainWindow" name="MissionCompilerMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="compileOptionsPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>776</width>
         <height>426</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Mission Compiler Options</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="5" column="4">
         <layout class="QVBoxLayout" name="addRemoveLayout">
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addSelectedButton">
            <property name="toolTip">
             <string>Add Selected</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mission_compiler.qrc">
              <normaloff>:/buttons/images/arrow-right-2.png</normaloff>:/buttons/images/arrow-right-2.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="addAllButton">
            <property name="toolTip">
             <string>Add All</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mission_compiler.qrc">
              <normaloff>:/buttons/images/arrow-right-double-2.png</normaloff>:/buttons/images/arrow-right-double-2.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="removeAllButton">
            <property name="toolTip">
             <string>Remove All</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mission_compiler.qrc">
              <normaloff>:/buttons/images/arrow-left-double-2.png</normaloff>:/buttons/images/arrow-left-double-2.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeSelectedButton">
            <property name="toolTip">
             <string>Remove Selected</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="mission_compiler.qrc">
              <normaloff>:/buttons/images/arrow-left-2.png</normaloff>:/buttons/images/arrow-left-2.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="5" column="3">
         <widget class="QListView" name="allPrimitivesList">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QListView" name="selectedPrimitivesList">
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="allPrimitivesLabel">
          <property name="text">
           <string>All Primitives</string>
          </property>
         </widget>
        </item>
        <item row="4" column="5">
         <widget class="QLabel" name="selectedPrimitivesLabel">
          <property name="text">
           <string>Selected Primitives</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" colspan="3">
         <widget class="QGroupBox" name="filterGroupBox">
          <property name="title">
           <string>Filter Criteria</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="dataDirLayout">
             <item>
              <widget class="QLabel" name="dataDirLabel">
               <property name="text">
                <string>Data Directory</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="dataDirEdit"/>
             </item>
             <item>
              <widget class="QToolButton" name="dataDirButton">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="filterLayout">
             <item>
              <widget class="QLabel" name="filterLabel">
               <property name="text">
                <string>Filter</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="filterEdit">
               <!-- Removed due to incompatibility with Qt 4.6 and uic still adding it to the source.
               <property name="placeholderText">
                <string>type filter here</string>
               </property> -->
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetFiltersButton">
               <property name="toolTip">
                <string>Blanks out the filter and reloads all files from the data directory.</string>
               </property>
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
          <zorder>dataDirLabel</zorder>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="publishOptionsPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>776</width>
         <height>426</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Publish Options</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListWidget" name="publishServersList"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="compilationOutputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>776</width>
         <height>426</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Compilation Output</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="compileOutputText">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionValidate"/>
   <addaction name="actionCompile"/>
   <addaction name="actionPublish"/>
  </widget>
  <action name="actionCompile">
   <property name="icon">
    <iconset resource="mission_compiler.qrc">
     <normaloff>:/buttons/images/run-build-2.png</normaloff>:/buttons/images/run-build-2.png</iconset>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
   <property name="toolTip">
    <string>Compile</string>
   </property>
  </action>
  <action name="actionValidate">
   <property name="icon">
    <iconset resource="mission_compiler.qrc">
     <normaloff>:/buttons/images/news-subscribe-2.png</normaloff>:/buttons/images/news-subscribe-2.png</iconset>
   </property>
   <property name="text">
    <string>Validate</string>
   </property>
   <property name="toolTip">
    <string>Validate</string>
   </property>
  </action>
  <action name="actionPublish">
   <property name="icon">
    <iconset resource="mission_compiler.qrc">
     <normaloff>:/buttons/images/document-export-4.png</normaloff>:/buttons/images/document-export-4.png</iconset>
   </property>
   <property name="text">
    <string>Publish</string>
   </property>
   <property name="toolTip">
    <string>Compile and Publish</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mission_compiler.qrc"/>
 </resources>
 <connections/>
</ui>
